data/human/human missions.txt

```diff
@@ -5426,3 +5426,335 @@ mission "Deal Change 2"
+			`	You freeze. Larry has your own sidearm in his hand, pointed at you. He must have taken it last night, when you thought they were partying. Instantly, all drunkenness is gone from him and his friends, and they eye you with sudden glee and hostility. "That's right," Howard says, with a snark. "Now, will you please let us into the cockpit?"`
+			`	You step away from the door, careful not to make any sudden moves.`
+				goto options
+			
+			label come
+			`	You let the group into the cockpit. Four of them immediately crowd the place, but the fifth teen - the one leaning on the wall - keeps his distance.`
+			`	The group seems intrigued by the various controls and equipment that are all over the cockpit. They ask you questions and walk around with clear curiosity. Then, all of a sudden, those inside grab you and try to pin your hands down.`
+			choice
+				`	(Call for help.)`
+				`	(Fight them.)`
+					goto fight
+				`	(Reach for my sidearm.)`
+					goto sidearm
+			`	You try to shout, but a kidney punch knocks the wind out of you before you can speak.`
+				goto fight
+
+			label sidearm
+			`	You try to reach for your pistol, but it's no longer in its holster.`
+
+			label fight
+			`	You stand no chance against the four attackers. They each grab one of your limbs and immobilize you as the remaining boy points your own gun at you. They must have taken it last night, when you thought they were partying.`
+			`	"Keep it steady, captain," Howard says with glee from your right side. "No funny business. Larry here has an itchy trigger finger." He then lets go of your right leg he has been holding onto, and addresses Larry. "Told you it would work! Next time, I get the gun while you fight in the dirt with the idiot." You notice how none of them seem to be drunk in the slightest anymore.`
+
+			label options
+			`	"You will fly the ship for us. If you don't try anything, we'll let you go afterwards." You don't give much credit to their promise.`
+			choice
+				`	(Pretend to comply with the demands.)`
+					goto comply
+				`	(Shout for your crew.)`
+
+			branch survived
+				random < "flagship crew" - 1
+			`	Larry fires the weapon, and you feel a sickening pain in your chest. You fall to the ground and try to gasp for air, but it feels too hard. You watch as everything goes black and your consciousness slips away.`
+				die
+
+			label survived
+			`	Larry fires the weapon, and you feel sickening pain in your left shoulder. Before Larry can fire again however, a member of your crew pops up behind him and takes him out with a well-aimed shot.`
+			`	With the other four attackers left unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and stitch up your wound. You still can't quite believe what just happened.`
+			`	You thank your fellow crew members for saving your life and decide to not let your sidearm out of your sight ever again.`
```

Comment:
```suggestion
			`	You thank your fellow crew members for saving your life and vow to never let your sidearm out of your sight ever again.`
```

Reply:
```suggestion
			`	You thank your fellow crew members for saving your life and vow never to let your sidearm out of your sight ever again.`
```

This seems a little more fluid, although it would be fine either way.